de-lib/src/main/java/org/iplantc/de/client/models/pipelines/PipelineTask.java

```diff
@@ -13,35 +13,42 @@
  */
 public interface PipelineTask extends HasName {
 
+    @PropertyName("system_id")
```

Comment:
I think we can remove these two additions and extend `HasSystemId` instead here like you did in `JobExecution.java`